webapp/src/modules/mapBuilder.js

```diff
@@ -173,9 +173,12 @@ export default class MapBuilder {
       .locate({
         watch: true,
         maxZoom: 20,
+        timeout: 20000,
         enableHighAccuracy: true,
       })
-      .on("locationerror", (e) => onGetLocationError(e))
+      .on("locationerror", (e) => {
+        if (e.code === (1 || 2)) return onGetLocationError();
```

Comment:
What about other codes? Also can you move 1 and 2 into named constants so it's clear what they represent?